### PR TITLE
Tinlake pools fixes

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

This pull request...

Closes #<issue-number>

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/components/TokenList.tsx [hunk 1]
```diff
@@ -94,19 +95,28 @@ export const TokenList: React.FC<Props> = ({ tokens }) => {
...
+            bottom={0}
+            left={0}
+            width={16}
+            height={16}
+            borderRadius="50%"
+            background="white"
+            style={{ transform: 'translate(-50%, 50%)' }}
+          >
+            <Box as="img" src={ethereumLogo} height={12} mx="auto" />
+          </Shelf>
```
Comment (Hornebom) [inline centrifuge-app/src/components/TokenList.tsx:114]: Same as above, right? Can you make this a component?

File: centrifuge-app/src/pages/Pool/Header.tsx [hunk 2]
```diff
@@ -31,24 +33,40 @@ export const PoolDetailHeader: React.FC<Props> = ({ actions }) => {
...
+              bottom={0}
+              left={0}
+              width={22}
+              height={22}
+              borderRadius="50%"
+              background="white"
+              style={{ transform: 'translate(-50%, 50%)' }}
+            >
+              <Box as="img" src={ethereumLogo} height={18} mx="auto" />
+            </Shelf>
```
Comment (Hornebom) [inline centrifuge-app/src/pages/Pool/Header.tsx:67]: make this a component

**Review by Hornebom — CHANGES_REQUESTED** [review]

File: centrifuge-app/src/assets/images/ethereum.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg width="13" height="23" viewBox="0 0 13 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g opacity="0.7">
```
[the remark this answers is not in the document]
Comment (onnovisser) [reply centrifuge-app/src/assets/images/ethereum.svg]: not sure why that was there, removed

File: centrifuge-app/src/components/TokenList.tsx (hunk 1 above)
Comment (onnovisser) [reply centrifuge-app/src/components/TokenList.tsx]: done

**Review by Hornebom — APPROVED** [review]

**Pull request merged**